packages/fiori/src/UploadCollectionItem.js

```diff
@@ -101,7 +101,7 @@ const metadata = {
 		 * @defaultvalue false
 		 * @public
 		 */
-		noTerminate: {
+		hideTerminate: {
```

Comment:
I think we agreed at the end to use hideRetryButton, hideTerminateButton, disableDeleteButton to make the props self explanatory

Reply:
Renamed the properties, added a new property `hideDeleteButton`. Added tests.